### PR TITLE
setup project

### PR DESCRIPTION
**Pull request by kienyk02**
## Related Tickets
- [#Task 78095: Content](https://edu-redmine.sun-asterisk.vn/issues/78095)
## WHAT
- 
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [x] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -9,10 +9,11 @@
...
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:text="@string/kvrevr"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
+
 </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (HVgiang86) [inline app/src/main/res/layout/activity_main.xml]:
```suggestion
</androidx.constraintlayout.widget.ConstraintLayout>

```

Comment (HVgiang86) [reply app/src/main/res/layout/activity_main.xml]: File content must end with a newline

File: app/ktlint.gradle [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    group = 'formatting'
+    description = 'Fix Kotlin code style deviations.'
+    classpath = configurations.ktlint
+    main = 'com.pinterest.ktlint.Main'
+    args '-F', 'src/**/*.kt'
+}
+
+dependencies {
+    ktlint "com.pinterest:ktlint:1.3.1"
+}
```
Comment (HVgiang86) [inline app/ktlint.gradle]:
```suggestion
}

```

Comment (HVgiang86) [reply app/ktlint.gradle]: File content must end with a newline

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">Android-template</string>
+    <string name="kvrevr">kvrevr</string>
 </resources>
```
Comment (HVgiang86) [inline app/src/main/res/values/strings.xml]:
```suggestion
</resources>

```

Comment (HVgiang86) [reply app/src/main/res/values/strings.xml]: File content must end with a newline

**Review by HVgiang86 — CHANGES_REQUESTED** [review]
- Không được xoá Pull Request description
- Các file luôn kết thúc bằng một dòng trống

**Review by huyenmi16 — APPROVED** [review]
- [HN_Pra19_Android]_Le.Ngoc.Huyen (T2-W, T3-W, T4-W, T5-W, T6-A) nhóm 4 đồng ý với thay đổi của bạn

**Review by nhat15072002 — APPROVED** [review]

File: app/ktlint.gradle (hunk 2 above)
Comment (ghost) [inline app/ktlint.gradle]:
```suggestion
}

```

Comment (ghost) [reply app/ktlint.gradle]: File content must end with a newline

**Review by ghost — CHANGES_REQUESTED** [review]
check my cmts

**Pull request merged**